product_docs/docs/epas/16/upgrading/index.mdx

```diff
@@ -8,6 +8,6 @@ redirects:
 
 Upgrading an EDB Postgres Advanced Server installation varies depending on whether you want to perform a major or minor version upgrade. 
 
--   Use `pg_upgrade` to perform upgrades between major versions of EDB Postgres Advanced Server. 
--   Use `dnf`, `apt-get` or StackBuilder Plus to perform upgrades between minor versions of EDB Postgres Advanced Server. The method you use depends on the underlying operating system. 
+-   Use pg_upgrade to perform upgrades between major versions of EDB Postgres Advanced Server. 
+-   Use the DNF software package manager, apt-get, or StackBuilder Plus to perform upgrades between minor versions of EDB Postgres Advanced Server. The method you use depends on the underlying operating system. 
```

Comment:
DNF and apt-get are not the same, but rather two different package managers. In the past, EDB used DNF, but now they have "updated" to using apt-get. 
I want to mention both here, because even though I am trying to update docs to use apt-get only, there might be some outdated references out there. Another alternative would be: 

```suggestion
-   Use the apt-get software package manager (formerly, EDB used DNF) or StackBuilder Plus to perform upgrades between minor versions of EDB Postgres Advanced Server. The method you use depends on the underlying operating system. 
```